### PR TITLE
Initial signup component

### PR DESCRIPTION
**Pull request by JessieFrance**
This PR aims to create an initial sign up component for #12.

### CODE REVIEW

File: src/Signup.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+const Signup = (): JSX.Element => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirm, setConfirm] = useState("");  
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    // Do something with name, email, password, and confirmed password.
```
Comment (ArturoGrullon) [inline src/Signup.tsx]: These console.logs will not be required in the production code

File: src/Signup.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+const Signup = (): JSX.Element => {
+  const [name, setName] = useState("");
```
Comment (ArturoGrullon) [inline src/Signup.tsx]: These inputs can be handled by one useState, [here](https://dev.to/deboragaleano/how-to-handle-multiple-inputs-in-react-55el) you can read more about it.

File: src/Signup.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  };
+
+  return (
+    <>
+	<h2>Sign Up</h2>
+	<form onSubmit={(e) => handleSubmit(e)}>
+            <label>
+		Name:
+		<input
+		  type="text"
```
Comment (ArturoGrullon) [inline src/Signup.tsx]: Would be nice to have a [placeholder](https://www.w3schools.com/tags/att_input_placeholder.asp).

File: src/Signup.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+		  name="name"
+		  value={name}
+		  onChange={(e) => setName(e.target.value)}
+		/>
+            </label>
+            <br />
+            <label>
+		Email:
+		<input
+		  type="email"
```
Comment (ArturoGrullon) [inline src/Signup.tsx]: Here as well

**Review by ArturoGrullon — CHANGES_REQUESTED** [review]
Overall it's pretty great, just requires minor changes that will just enhance a little bit the code.

Great job 😄

**Comment by rmcguire6** [comment]
I agree with Arthur's changes.

**Comment by JessieFrance** [comment]
Thank you @ArturoDucasse and @rmcguire6 for the feedback on this. The changes have been made.

**Review by rmcguire6 — APPROVED** [review]
Looks great, Jessie.

**Review by ArturoGrullon — APPROVED** [review]
Nice, now is slightly better than it already was.

**Pull request merged**